auditwheel/repair.py

```diff
@@ -1,22 +1,44 @@
 import os
+import re
 import shutil
 import itertools
+import functools
 from os.path import exists, relpath, dirname, basename, abspath, isabs
 from os.path import join as pjoin
-from subprocess import check_call
+from subprocess import check_call, check_output, CalledProcessError
 from distutils.spawn import find_executable
 from typing import Optional
 
 from .policy import get_replace_platforms
 from .wheeltools import InWheelCtx, add_platforms
 from .wheel_abi import get_wheel_elfdata
-from .elfutils import elf_read_rpaths, is_subdir
+from .elfutils import elf_read_rpaths, is_subdir, elf_read_dt_needed
+from .hashfile import hashfile
 
 
-def repair_wheel(wheel_path: str, abi: str, lib_sdir: str, out_dir:
-                 str, update_tags: bool) -> Optional[str]:
+@functools.lru_cache()
+def verify_patchelf():
+    if not find_executable('patchelf'):
+        raise ValueError('Cannot find required utility `patchelf` in PATH')
+    try:
+        version = check_output(['patchelf', '--version']).decode('utf-8')
+    except CalledProcessError:
+        raise ValueError('Could not call `patchelf` binary')
+
+    m = re.match('patchelf\s+(\d+(.\d+)?)', version)
+    if m:
+        if float(m.group(1)) >= 0.9:
+            return True
```

Comment:
This can never return anything else, right? I that case I think a simple `return` (without `True`) would be enough.